.github/workflows/ngsast-docker.yml

```diff
@@ -0,0 +1,42 @@
+# This workflow integrates ShiftLeft Inspect with GitHub
```

Comment:
Perhaps we rename this to ngsast-docker.yml.example? Otherwise we will performing two analysis per PR.